changelogs/fragments/502_postgresql_privs.yml

```diff
@@ -0,0 +1,2 @@
+major_changes:
+- "postgresql_privs - Improved roles management (https://github.com/ansible-collections/community.postgresql/pull/502)."
```

Comment:
Please elaborate. The changelog should describe all changes in the behavior and interface of the module. Only refactoring can be omitted here.